src/main/java/mekanism/common/item/gear/ItemMekaTool.java

```diff
@@ -342,18 +360,50 @@ public boolean canApplyAtEnchantingTable(ItemStack stack, Enchantment enchantmen
         return false;
     }
 
-    @Override
-    public boolean supportsSlotType(ItemStack stack, @Nonnull EquipmentSlot slotType) {
-        return IModeItem.super.supportsSlotType(stack, slotType) && getModules(stack).stream().anyMatch(Module::handlesModeChange);
-    }
+    public enum MekaToolMode implements IRadialSelectorEnum<MekaToolMode>, IHasTextComponent {
+        ONE("mode_1"),
+        TWO("mode_2"),
+        THREE("mode_3"),
+        FOUR("mode_4"),
+        FIVE("mode_5"),
+        SIX("mode_6");
```

Comment:
In the state that this enum currently is, I feel like the `key` parameter shouldn't exist, and `getKey` should just return `"mode_" + (ordinal() + 1)`.

Reply:
Valid point, I added it because I was constructing it in multiple places but it doesn't have to be defined I guess.